pandas/core/arrays/base.py

```diff
@@ -14,12 +14,15 @@ class ExtensionArray(object):
     with a custom type and will not attempt to coerce them to objects. They
     may be stored directly inside a :class:`DataFrame` or :class:`Series`.
 
+    .. versionadded:: 0.23.0
```

Comment:
Can you add a note about the experimental status in this file as well (eg in module docstring at line 1).